examples/ex_clustered_inverse_mne/main.cpp

```diff
@@ -293,7 +293,7 @@ int main(int argc, char *argv[])
 
     p3DDataModel->addSurfaceSet(parser.value(subjectOption), "MRI", t_surfSet, t_annotationSet);
 
-    if(MneEstimateTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
+    if(MneDataTreeItem* pRTDataItem = p3DDataModel->addSourceData(parser.value(subjectOption),
```

Comment:
Mind indentation

Reply:
Sorry, don't see what is wrong here.

Reply:
The passed functions parameter are seperated by linebreaks and therefore should be indented to start at the function's bracket `(`